bundler.gemspec

```diff
@@ -1,7 +1,13 @@
 # coding: utf-8
 # frozen_string_literal: true
 
-require File.expand_path("../lib/bundler/version", __FILE__)
+begin
+  require File.expand_path("../lib/bundler/version", __FILE__)
+rescue LoadError
+  # for Ruby core repository
+  require File.expand_path("../bundler/version", __FILE__)
```

Comment:
Is there a way to avoid this? Using a `rescue` to support Ruby core does not seem nice.

Reply:
we could use `File.file?`

Reply:
`lib/bundler/version` string was used the other example directory. This string can be only once in gemspec.

Reply:
see https://github.com/bundler/bundler/blob/master/spec/runtime/setup_spec.rb#L877